resources/scripts/components/dashboard/forms/SetupTwoFactorModal.tsx

```diff
@@ -75,41 +86,50 @@ const SetupTwoFactorModal = () => {
                     .matches(/^(\d){6}$/, 'Authenticator code must be 6 digits.'),
             })}
         >
-            {recoveryTokens.length > 0 ?
+            {recoveryTokens.length > 0 ? (
                 <>
                     <h2 css={tw`text-2xl mb-4`}>Two-factor authentication enabled</h2>
                     <p css={tw`text-neutral-300`}>
-                        Two-factor authentication has been enabled on your account. Should you loose access to
-                        this device you&apos;ll need to use one of the codes displayed below in order to access your
-                        account.
+                        Two-factor authentication has been enabled on your account. Should you loose access to this
+                        device you&apos;ll need to use one of the codes displayed below in order to access your account.
                     </p>
                     <p css={tw`text-neutral-300 mt-4`}>
-                        <strong>These codes will not be displayed again.</strong> Please take note of them now
-                        by storing them in a secure repository such as a password manager.
+                        <strong>These codes will not be displayed again.</strong> Please take note of them now by
+                        storing them in a secure repository such as a password manager.
                     </p>
                     <pre css={tw`text-sm mt-4 rounded font-mono bg-neutral-900 p-4`}>
-                        {recoveryTokens.map(token => <code key={token} css={tw`block mb-1`}>{token}</code>)}
+                        {recoveryTokens.map((token) => (
+                            <code key={token} css={tw`block mb-1`}>
+                                {token}
+                            </code>
+                        ))}
                     </pre>
                     <div css={tw`text-right`}>
                         <Button css={tw`mt-6`} onClick={dismiss}>
                             Close
                         </Button>
                     </div>
                 </>
-                :
+            ) : tokenUrl && tokenUrl.length ? (
```

Comment:
Please don't change the way this modal works, just leave it how it was and show the token when it is available in the state. We don't need a spinner to show up here, just keep it how it currently works.

Reply:
Okay.